src/components/MessageItem/MessageItem.jsx

```diff
@@ -33,6 +32,13 @@ const MessageItem = ({
   const firstLetterOfName = userName.substr(0, 1).toUpperCase();
   const isCurrentUser = userId === currentUserId;
 
+  if (
+    type === MESSAGE_TYPES.START_TYPING ||
```

Comment:
```suggestion
    [MESSAGE_TYPES.START_TYPING, MESSAGE_TYPES.STOP_TYPING].includes(type)
```
I just don't like those operators ||